rholang/src/test/scala/coop/rchain/rholang/interpreter/RegistrySpec.scala

```diff
@@ -608,32 +606,31 @@ class RegistrySpec extends FlatSpec with Matchers with RegistryTester {
       List(Channel(Quote(GString(s))))
 
     val expectedBundle: Par = Bundle(GPrivate(
-      ByteString.copyFrom(
-        Base16.decode("2abed4680a1dca886783d04584f8b9b600f50848c8192e31e63d89828f635781"))),
+      ByteString.copyFrom(registeredName)),
       writeFlag = true,
       readFlag = false)
 
+    // Mimic the uri creation
+    val fullKey = new Array[Byte](34)
+    Array.copy(uriBytes, 0, fullKey, 0, 32)
+    val crc = Registry.CRC14.compute(fullKey.view.slice(0, 32))
+    fullKey(32) = (crc & 0xff).toByte
+    fullKey(33) = ((crc & 0xff00) >>> 6).toByte
+    val expectedUri = "rho:id:" + ZBase32.encodeToString(fullKey, 270)
```

Comment:
Let's extract this code to a helper method `createUri(bytes: Array[Byte]): GUri` and use here and in the `Registry.`